juju/model.py

```diff
@@ -2108,23 +2108,17 @@ def resolve(self, reference):
             will be 0 and separate AddUnitChanges will be used.  For Kubernetes
             models, this will be used to scale the application.
             (Only given on Juju 2.5+)
-
-        :param placement string:
-            Placement holds the placement directive for units of this
-            application.  Only applicable for Kubernetes models.
-            (Only given on Juju 2.5+)
         """
         # resolve indirect references
         charm = self.resolve(charm)
 
         if len(args) == 1:
             # Juju 2.4 and below only sends the resources
             resources = args[0]
-            devices, num_units, placement = None, None, None
+            devices, num_units = None, None, None
         else:
             # Juju 2.5+ sends devices before resources, as well as num_units
-            # and placement
-            devices, resources, num_units, placement = args
+            devices, resources, num_units = args
```

Comment:
Hasn't this just broken deploys on 2.5.0 and 2.5.1?

Reply:
Not anymore :-)